Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'acts-as-taggable-on', '~> 6.0'
 gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
 gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'
 gem 'sendgrid-ruby'
+gem 'traceroute'
```

Comment:
super simple. Just ran `rake traceroute` and saw unused routes as well as a big ass list of unreachable action methods, which were almost entirely application_controller methods that were unnecessarily inherited everywhere.

Reply:
Nice! that probably will increase our spec coverage